File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //prompt for input
    int height;
    do
 {
        height=get_int("height");
 }
    while (height<1 || height>8);

for (int row=0; row<height; row++)
{
     for(int spaces=height-1; spaces>row; spaces--)
    {
        printf(" ");
    }
    
    for(int hashes=row+height+1 ;hashes>height; hashes--)
    {
        printf("#");
    }

    printf("\n");
}
}

File: pset6/mario.py
from cs50 import get_int

while True:
    # prompt for input
    height = get_int("Height: ")
    # reject if smaller than 1 or greater than 8
    if height < 1 or height > 8:
        height = get_int("Height: ")
    # accept if in range
    if height >= 1 and height <= 8:
        break

# go row by row
for row in range(height):
    # print spaces
    print((height-row-1) * " ", end="")
    # print #
    print((row+1) * "#")